bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -42,7 +42,9 @@ def __init__(self, kms_providers, schema_map=None):
               - `aws`: Map with "accessKeyId" and "secretAccessKey" as strings.
               - `azure`: Map with "clientId" and "clientSecret" as strings.
               - `gcp`: Map with "email" and "privateKey" as strings.
```

Comment:
The spec says that  ["privateKey" can also be a `bytes` or base64 encoded string](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#kmsproviders):
```
gcp: {
   email: String,
   privateKey: byte[] or String, // May be passed as a base64 encoded string.
   endpoint: Optional<String> // Defaults to oauth2.googleapis.com
}
```

I think we should test this behavior as well.

Reply:
Documented and tested!